### DISCUSSION TITLE
2022/06/trr-pure-edition-rom-samsung-j7-prime

### DISCUSSION THREAD

**Discussion by tegursapa** [madi10/MANTANKODE#13] [General]
# 2022/06/trr-pure-edition-rom-samsung-j7-prime

Masih menggunakan Samsung J7 Prime, berasa penyimpanan penuh, tampilan yang jadul, pingin suasana baru tapi stabil? Solusinya instal rom TRR.

https://www.mantankode.net/2022/06/trr-pure-edition-rom-samsung-j7-prime.html?m=1#toc-7

**Comment by madi10** [comment]
Yang bagus dan diakuin lancar dan aman. Yaitu di versi "TRR Pure 15"

**Reply by madi10** [reply]
Sudah coba TRR 14 dan bug nya lumayan banyak. Masih aman untuk digunakan yang versi 15.

**Comment by madi10** [comment]
Banyak yang nanya, bang kalau video call di whatsapp, bagus aja kah? Tidak gepeng? Tidak ke perbesar videonya?

Nah ini saya liat kan hasil vc nya.

![Screenshot_20221113-163746_WhatsAppBusiness.jpg](https://user-images.githubusercontent.com/32736268/202061613-7e776db3-1450-4e07-b9b2-99b97506f224.jpg)

![Screenshot_20221113-163733_WhatsAppBusiness.jpg](https://user-images.githubusercontent.com/32736268/202061653-a1dc1f5a-ab79-4463-89b0-0307880ac880.jpg)

**Comment by whoim24** [comment]
Gk perlu Format data kah bg?

**Reply by madi10** [reply]
perlu format data bang, jadi data nya yang ada di pindahkan ke tempat aman.

seperti di fd atau sejenisnya.
